bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -19,7 +19,7 @@ <h2 id="security-vulnerability-bounty">Security Vulnerability Bounty</h2>
   <p>Mozilla will pay a bounty for client security bugs as detailed below. All security bugs must follow the following general criteria to be eligible:</p>
 
   <ul class="mzp-u-list-styled">
-    <li>Eligible security bugs may be present in any of the current main development or released versions of Firefox, Firefox for Android, or Firefox for iOS as released by Mozilla Corporation (e.g. Nightly mozilla-central or Beta test versions, as well as the final release product versions).</li>
+    <li>Eligible security bugs may be present in any of the current main development or released versions of Firefox, Firefox for Android, or Firefox for iOS as released by Mozilla Corporation (e.g. Nightly mozilla-central or Beta test versions, as well as the final release product versions). End-of-life products are not covered.</li>
```

Comment:
I could go with this, but I'd prefer a more explicit "End-of-life products are not eligible for a security bug bounty."

Not part of this change, but in case you want to address it here rather than have me file a separate issue for it:  We all know "current" is supposed to apply to the "released version" clause, but someone could argue it's "current dev" and "(any) released versions". That's a dumb argument obviously not in good faith but if we tweak it slightly we can avoid it:  "current main development or most recent released versions of ...". It's the interposition of "main" in there that makes me mentally bind "current main" to "development" only, whereas without "main" I tend to see it as intended "current (development or released) versions".  We added "main" was to forestall folks claiming bounties on experimental branches so we probably don't want to simply drop it.

We might not need "current" before "main development" at all: move it to just before "released". If they're testing an outdated version of the main development branches they're not going to get bounties if we've already fixed the bugs.

English would be a lot less ambiguous if we were allowed to use mathematical parentheses.